src/drivers/pwm_out_sim/CMakeLists.txt

```diff
@@ -30,13 +30,20 @@
 # POSSIBILITY OF SUCH DAMAGE.
 #
 ############################################################################
+
+if (${PX4_BOARD_VENDOR}_${PX4_BOARD_MODEL} STREQUAL "px4_sitl")
+	set(module_config "module_sim.yaml")
```

Comment:
I'm not sure what to think about the HITL/SITL split. Realistically couldn't it be HIL_ACT for both and then we can share simulation airframes (gazebo)?

Reply:
Yes we have several options here, I don't have a strong preference. I didn't see this as final here either, more as something that generally works.